.circleci/config.yml

```diff
@@ -97,8 +98,44 @@ jobs:
       - store_artifacts:
           path: results
 
+  coverage:
+    docker:
+      - image: cimg/python:3.12
+    steps:
+      - checkout
+      - run:
+          name: Before install
+          command: |
+            sudo apt-get update
+            sudo apt-get install graphviz graphviz-dev
+      - run:
+          name: Install dependencies
+          command: |
+            python -m venv venv
+            source venv/bin/activate
+            pip install --upgrade pip
+            pip install .[default,extra,test]
+            pip install coverage
+            pip list
+      - run:
+          name: Run tests
+          command: |
+            source venv/bin/activate
+            pytest --cov=networkx --cov-report=html --runslow --doctest-modules --durations=20 --pyargs networkx
+      - run:
+          name: Generate coverage report
+          command: |
+            source venv/bin/activate
+            coverage html
```

Comment:
Testing locally, I *think* we can get away with this step too - my understanding is the `--cov-report=html` flag does this, though I'm not 100% sure

Reply:
Seems like it, good catch!